xbanish.c

```diff
@@ -109,6 +111,10 @@ main(int argc, char *argv[])
 					ignored |= mods[i].mask;
 
 			break;
+        case 'k':
```

Comment:
The indentation seems messed up, ensure you're using TABs.

Reply:
Also fixed. I forgot to change my editor config. I added an .editorconfig file just for me 😄